data.sql

```diff
@@ -38,3 +38,34 @@ UPDATE ANIMALS SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE ANIMALS SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE ANIMALS SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
 UPDATE ANIMALS SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+INSERT INTO VETS (name, age, date_of_graduation)
+VALUES ('Vet William Tatcher', 45, 'Apr 23 2000'),
+('Vet Maisy Smith', 26 ,'Jan 17 2019'),
+('Vet Stephanie Mendez', 64, 'May 4 1981'),
+('Vet Jack Harkness', 38,'Jun 8 2008');
+
+INSERT INTO SPECIALIZATIONS (species_id, vets_id) VALUES (1,1),(1,3),(2,3),(2,4);
```

Comment:
- [ ] [_Optional_] Alternatively, you can search for the correct id using the following commands
``` SQL
INSERT INTO specializations (species_id, vet_id)
 VALUES ( (SELECT id FROM species WHERE name = 'Pokemon'), (SELECT id FROM vets WHERE name = 'William Tatcher'));

```